src/EditorFeatures/Core/Shared/Utilities/IUIContextActivationService.cs

```diff
@@ -0,0 +1,15 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+
+namespace Microsoft.CodeAnalysis.Editor.Shared.Utilities;
+
+internal interface IUIContextActivationService
```

Comment:
Editor features is not available in vscode.  If you're expecting the code in the EA layer to run in vscode, this needs to be moved down, to the protocol layer or features layer.

Reply:
As mentioned, this doesn't ship in VS Code. We'll need a new EA assembly for that eventually.